deploy/docker/utils/bin/utils.js

```diff
@@ -73,10 +73,32 @@ async function listLocalBackupFiles(){ // Ascending order
   return backupFiles;
 }
 
+
+async function updateLastBackupErrorMailSentInMilliSec(ts) {
+  try {
+    await fsPromises.access(Constants.BACKUP_PATH);
+  } catch (error) {
+    await fsPromises.mkdir(Constants.BACKUP_PATH);
+  }
```

Comment:
This can actually be simplified with the `recursive` option. From the docs:

> Calling fsPromises.mkdir() when path is a directory that exists results in a rejection only when recursive is false.
> -- https://nodejs.org/api/fs.html#fspromisesmkdirpath-options

```suggestion
  await fsPromises.mkdir(Constants.BACKUP_PATH, { recursive: true });
```